extensions/amp-lightbox/validator-amp-lightbox.protoascii

```diff
@@ -58,3 +58,32 @@ tags: {  # <amp-lightbox>
     supported_layouts: NODISPLAY
   }
 }
+tags: { # <amp-lightbox close-button> in A4A.
+  html_format: AMP4ADS
+  html_format: EXPERIMENTAL
+  tag_name: "AMP-LIGHTBOX"
```

Comment:
`tag_name` is unique except when there is also a `spec_name`. Perhaps `spec_name: "amp-lightbox [a4a]"` or amp4ads instead of a4a. also, why are both `html_format: EXPERIMENTAL`?

@Gregable since I'm out tomorrow.

Reply:
Done.

Not sure why the original `tag` is marked allowed in experimental.